sunbeam-python/sunbeam/features/vault/feature.py

```diff
@@ -348,6 +351,24 @@ def run(self, status: Status | None = None) -> Result:
                     if len(non_leader_units) == 0:
                         message = "Vault unseal operation status: completed"
                     else:
+                        # Wait for leader vault to update non-leader vault units
```

Comment:
What happens if the leader gets sealed, then the human operator unseals the leader using the comands?  (but the followers remained unsealed the whole)?

Reply:
Not sure if there's a way only the leader can get sealed?, So this might not be a valid scenario anyway?

Reply:
Added check to see if status message is empty

Reply:
It is a valid scenario where leader unit pod can be rescheduled/deleted. Added new check to handle this case.